broadinstitute/pearl
# [JN-512] adding new ourhealth team to about us

Adding new folks to about us page.  This doesn't complete the ticket as there is someone to be added that we don't have a bio or pic for, but good to get in the ones we have.

TO TEST:
1. repopulate ourhealth
2. go to `https://sandbox.ourhealth.localhost:3001/aboutUs` 
3. confirm new people show up
